.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,11 +1,10 @@
 ### Description
 
-[Please describe what this change achieves]
+Adds the ability to specify a certificate stored in Azure KeyVault and add to an Azure VM.
```

Comment:
Please could you remove this file from your PR? The files in .github/ are internal helpers for the GitHub UI

Reply:
Sure thing.